go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/ignite/cli
 go 1.18
 
 require (
+	cosmossdk.io/math v1.0.0-beta.3
```

Comment:
I'm curious to understand how it's possible to have a version `v1` while no such git tag exists in the cosmos-sdk repo. 

I mean I know the concept of canonical import, but I wasn't aware it could be used to mimic tags.

Reply:
@tbruyelle I think what you are looking for is the `math/v1.0.0-beta.3` branch which is the one that assigns the "v1.0.0-beta.3" version to the `math` package

Reply:
Oh indeed thanks! Actually it's not a branch but a tag with the `math/` prefix! Cool this is how it works then, interesting.

Sounds a little tricky that when you run `go get cosmossdk.io/math`, the `math` can interpreted as a path or a part of the tag name...

Reply:
This is actually a common practice, like in golang.org/x/tools via their vcs https://cs.opensource.google/go/x/tools, but not really documented afaik.